production/loki-mixin/README.md

```diff
@@ -0,0 +1,44 @@
+# loki-mixin
+
+loki-mixin is a jsonnet library containing a set of loki monitoring dashboards, alerts and rules used on our production setup. 
+
+## Dashboards
+
+To test the dashboards against a local grafana & loki setup perform the following steps.
+
+### Pre-requisites
+
+* Grizzly is a tool for managing jsonnet dashboards in Grafana: https://github.com/grafana/grizzly.
+To install it follow the instructions at: https://grafana.github.io/grizzly/installation/.
+
+* Make sure you have the latest dependencies in the `vendor` directory by running the following command in `production/loki-mixin`:
+
+```shell
+jb install
+```
+
+* On your Grafana instance create an API key with role 'Admin' under Configuration > API keys. 
+Copy this key for the next step.
+
+### Testing dashboards
+
+To test the dashboard in your local grafana instance, in directory `production/loki-mixin` run the command:
+
+```shell
+GRAFANA_URL=http://localhost:3000 GRAFANA_TOKEN=<API_KEY> grr watch ./ dashboards.libsonnet
```

Comment:
Do we also need to set the `JSONNET_PATH` env variable so files within the `vendor/` (and folder are recognized?

Reply:
Good catch, done in: https://github.com/grafana/loki/pull/6083/commits/b72fc8fb1fae12df57fef308ec1b8c9b4671394c